windows/build_release.bat

```diff
@@ -1,13 +0,0 @@
-setlocal
```

Comment:
I usually use this batch for local testing. Is there a way to execute (part of) the azure-scripts instead?

Reply:
I'm not sure about that. Let me defer to the professionals. @kinke, @thewilsonator, @CyberShadow. Can you provide a response to this question? Thanks.

Reply:
> Is there a way to execute (part of) the azure-scripts instead?

Not that I know of.

I found it better to have all logic in the YAML, where the directories for PATH are set up appropriately (e.g., I had no idea that the `dm` dir is supposed to be the DMC dir augmented by implib.exe).